<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Chris J. Dean</title> <meta name="author" content="Chris J. Dean"/> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."/> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"/> <link rel="shortcut icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>⚛️</text></svg>"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://cdeanj.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Chris </span>J. Dean</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="dean_tychus:_2018" class="col-sm-8"> <div class="title">Tychus: a whole genome sequencing pipeline for assembly, annotation and phylogenetics of bacterial genomes</div> <div class="author"> Christopher Dean, Noelle Noyes, Steven M. Lakin, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Pablo Rovira-Sanz, Xiang Yang, Keith Belk, Paul Morley, Richard Meinersmann, Zaid Abdo' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em> Mar 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Summary: Tychus is a tool that allows researchers to perform massively parallel whole genome sequence (WGS) analysis with the goal of producing a high confidence and comprehensive description of the bacterial genome. Key features of the Tychus pipeline include the assembly, annotation, alignment, variant discovery and phylogenetic inference of large numbers of WGS isolates in parallel using open-source bioinformatics tools and virtualization technology. All prerequisite tools and dependencies come packaged together in a single suite that can be easily downloaded and installed on Linux and Mac operating systems. Availability: Tychus is freely available as an open-source package under the MIT license, and can be down-loaded via GitHub (https://github.com/Abdo-Lab/Tychus).</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="doster_investigating_2018" class="col-sm-8"> <div class="title">Investigating Effects of Tulathromycin Metaphylaxis on the Fecal Resistome and Microbiome of Commercial Feedlot Cattle Early in the Feeding Period</div> <div class="author"> Enrique Doster, Pablo Rovira, Noelle R. Noyes, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Brandy A. Burgess, Xiang Yang, Margaret D. Weinroth, Steven M. Lakin, Christopher J. Dean, Lyndsey Linke, Roberta Magnuson, Kenneth I. Jones, Christina Boucher, Jamie Ruiz, Keith E. Belk, Paul S. Morley' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em> Jul 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The objective was to examine effects of treating commercial beef feedlot cattle with therapeutic doses of tulathromycin, a macrolide antimicrobial drug, on changes in the fecal resistome and microbiome using shotgun metagenomic sequencing. Two pens of cattle were used, with all cattle in one pen receiving metaphylaxis treatment (800 mg subcutaneous tulathromycin) at arrival to the feedlot, and all cattle in the other pen remaining unexposed to parenteral antibiotics throughout the study period. Fecal samples were collected from 15 selected cattle in each group just prior to treatment (Day 1), and again 11 days later (Day 11). Shotgun sequencing was performed on isolated metagenomic DNA, and reads were aligned to a resistance and a taxonomic database to identify alignments to antimicrobial resistance (AMR) gene accessions and microbiome content. Overall, we identified AMR genes accessions encompassing 9 classes of AMR drugs and encoding 24 unique AMR mechanisms. Statistical analysis was used to identify differences in the resistome and microbiome between the untreated and treated groups at both timepoints, as well as over time. Based on composition and ordination analyses, the resistome and microbiome were not significantly different between the two groups on Day 1 or on Day 11. However, both the resistome and microbiome changed significantly between these two sampling dates. These results indicate that the transition into the feedlot—and associated changes in diet, geography, conspecific exposure, and environment—may exert a greater influence over the fecal resistome and microbiome of feedlot cattle than common metaphylactic antimicrobial drug treatment.</p> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="lakin_hierarchical_2019" class="col-sm-8"> <div class="title">Hierarchical Hidden Markov models enable accurate and diverse detection of antimicrobial resistance sequences</div> <div class="author"> Steven M. Lakin, Alan Kuhnle, Bahar Alipanahi, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Noelle R. Noyes, Chris Dean, Martin Muggli, Rob Raymond, Zaid Abdo, Mattia Prosperi, Keith E. Belk, Paul S. Morley, Christina Boucher' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Communications Biology</em> Aug 2019 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The characterization of antimicrobial resistance genes from high-throughput sequencing data has become foundational in public health research and regulation. This requires mapping sequence reads to databases of known antimicrobial resistance genes to determine the genes present in the sample. Mapping sequence reads to known genes is traditionally accomplished using alignment. Alignment methods have high specificity but are limited in their ability to detect sequences that are divergent from the reference database, which can result in a substantial false negative rate. We address this shortcoming through the creation of Meta-MARC, which enables detection of diverse resistance sequences using hierarchical, DNA-based Hidden Markov Models. We first describe Meta-MARC and then demonstrate its efficacy on simulated and functional metagenomic datasets. Meta-MARC has higher sensitivity relative to competing methods. This sensitivity allows for detection of sequences that are divergent from known antimicrobial resistance genes. This functionality is imperative to expanding existing antimicrobial gene databases., Steven Lakin et al. present Meta-MARC, a computational method for identifying antimicrobial resistance sequences using DNA-based Hidden Markov Models. Because of its increased sensitivity, Meta-MARC is able to detect novel antimicrobial resistance sequences that are divergent from all known sequences.</p> </div> </div> </div> </li></ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="doster_megares_2020" class="col-sm-8"> <div class="title">MEGARes 2.0: a database for classification of antimicrobial drug, biocide and metal resistance determinants in metagenomic sequence data</div> <div class="author"> Enrique Doster, Steven M. Lakin, Christopher J. Dean, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Cory Wolfe, Jared G. Young, Christina Boucher, Keith E. Belk, Noelle R. Noyes, Paul S. Morley' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Nucleic Acids Research</em> Jan 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Abstract. Antimicrobial resistance (AMR) is a threat to global public health and the identification of genetic determinants of AMR is a critical component to e</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="slizovskiy_mobilization_2020" class="col-sm-8"> <div class="title">Mobilization of Antibiotic Resistance: Are Current Approaches for Colocalizing Resistomes and Mobilomes Useful?</div> <div class="author"> Ilya B. Slizovskiy, Kingshuk Mukherjee, Christopher J. Dean, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Christina Boucher, Noelle R. Noyes' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Frontiers in Microbiology</em> Jan 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Antimicrobial resistance (AMR) poses a global human and animal health threat, and predicting AMR persistence and transmission remains an intractable challenge. Shotgun metagenomic sequencing can help overcome this by enabling characterization of AMR genes within all bacterial taxa, most of which are uncultivatable in laboratory settings. Shotgun sequencing, therefore, provides a more comprehensive glance at AMR ‘potential’ within samples, i.e., the “resistome”. However, the risk inherent within a given resistome is predicated on the genomic context of various AMR genes, including their presence within mobile genetic elements (MGEs). Therefore, resistome risk stratification can be advanced if AMR profiles are considered in light of the flanking mobilizable genomic milieu (e.g., plasmids, integrative conjugative elements (ICEs), phages, and other MGEs). Because such mediators of horizontal gene transfer (HGT) are involved in uptake by pathogens, investigators are increasingly interested in characterizing that resistome fraction in genomic proximity to HGT mediators, i.e., the “mobilome”; we term this “colocalization”. We explored the utility of common colocalization approaches using alignment- and assembly-based techniques, on clinical (human) and agricultural (cattle) fecal metagenomes, obtained from antimicrobial use trials. Ordination revealed that tulathromycin-treated cattle experienced a shift in ICE and plasmid composition vs. untreated animals, though the resistome was unaffected during the monitoring period. Contrarily, the human resistome and mobilome composition both shifted shortly after antimicrobial administration, though this rebounded to pre-treatment status. Bayesian networks revealed statistical AMR-MGE co-occurrence in 19% and 2% of edges from the cattle and human networks respectively, suggesting a putatively greater mobility potential of AMR in cattle feces. Conversely, using Mobility Index (MI) and overlap analysis, abundance of de novo-assembled contigs supporting resistomes flanked by MGE increased shortly post-exposure within human metagenomes, though \textgreater40 days after peak dose such contigs were rare (~2 %). MI was not substantially altered by antimicrobial exposure across all cattle metagenomes, ranging 0.5–4.0%. We highlight that current alignment- and assembly-based methods estimating resistome mobility yield contradictory and incomplete results, likely constrained by approach-specific data inputs, and bioinformatic limitations. We discuss recent laboratory and computational advancements that may enhance resistome risk analysis in clinical, regulatory, and commercial applications.</p> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="dean_investigating_2021" class="col-sm-8"> <div class="title">Investigating the cow skin and teat canal microbiomes of the bovine udder using different sampling and sequencing approaches</div> <div class="author"> C.J. Dean, I.B. Slizovskiy, K.K. Crone, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'A.X. Pfennig, B.J. Heins, L.S. Caixeta, N.R. Noyes' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Journal of Dairy Science</em> Jan 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="dean_what_2021" class="col-sm-8"> <div class="title">What is the Microbiome and Why is it Important for Organic Livestock Production?</div> <div class="author"> Chris Dean, Felipe Pena-Mosca, Tui Ray, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Bradley Heins, Pablo Pinedo, Vinicius Machado, Luciano Caixeta, Noelle Noyes' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>eOrganic</em> Jan 2021 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="weinroth_considerations_2022" class="col-sm-8"> <div class="title">Considerations and best practices in animal science 16S ribosomal RNA gene sequencing microbiome studies</div> <div class="author"> Margaret D Weinroth, Aeriel D Belk, Chris Dean, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Noelle Noyes, Dana K Dittoe, Michael J Rothrock, Steven C Ricke, Phillip R Myer, Madison T Henniger, Gustavo A Ramírez, Brian B Oakley, Katie Lynn Summers, Asha M Miles, Taylor B Ault-Seay, Zhongtang Yu, Jessica L Metcalf, James E Wells' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Journal of Animal Science</em> Feb 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Microbiome studies in animal science using 16S rRNA gene sequencing have become increasingly common in recent years as sequencing costs continue to fall and bioinformatic tools become more powerful and user-friendly. The combination of molecular biology, microbiology, microbial ecology, computer science, and bioinformatics—in addition to the traditional considerations when conducting an animal science study—makes microbiome studies sometimes intimidating due to the intersection of different fields. The objective of this review is to serve as a jumping-off point for those animal scientists less familiar with 16S rRNA gene sequencing and analyses and to bring up common issues and concerns that arise when planning an animal microbiome study from design through analysis. This review includes an overview of 16S rRNA gene sequencing, its advantages, and its limitations; experimental design considerations such as study design, sample size, sample pooling, and sample locations; wet lab considerations such as field handing, microbial cell lysis, low biomass samples, library preparation, and sequencing controls; and computational considerations such as identification of contamination, accounting for uneven sequencing depth, constructing diversity metrics, assigning taxonomy, differential abundance testing, and, finally, data availability. In addition to general considerations, we highlight some special considerations by species and sample type.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="ray_microbiome_2022" class="col-sm-8"> <div class="title">The microbiome of common bedding materials before and after use on commercial dairy farms</div> <div class="author"> Tui Ray, Tara Nath Gaire, Christopher J. Dean, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Sam Rowe, Sandra M. Godden, Noelle R. Noyes' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Animal Microbiome</em> Mar 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Bovine mastitis is one of the most economically important diseases affecting dairy cows. The choice of bedding material has been identified as an important risk factor contributing to the development of mastitis. However, few reports examine both the culturable and nonculturable microbial composition of commonly used bedding materials, i.e., the microbiome. Given the prevalence of nonculturable microbes in most environments, this information could be an important step to understanding whether and how the bedding microbiome acts as a risk factor for mastitis. Therefore, our objective was to characterize the microbiome composition and diversity of bedding material microbiomes, before and after use.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="dean_evaluation_2022" class="col-sm-8"> <div class="title">Evaluation of Contamination in Milk Samples Pooled From Independently Collected Quarters Within a Laboratory Setting</div> <div class="author"> Chris J. Dean, Felipe Peña-Mosca, Tui Ray, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Bradley J. Heins, Vinicius S. Machado, Pablo J. Pinedo, Luciano S. Caixeta, Noelle R. Noyes' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Frontiers in Veterinary Science</em> Mar 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The primary objective of this observational study was to evaluate the prevalence of contamination from independently collected quarter-level milk samples pooled in a laboratory and subjected to bacterial culture. To address this objective, weekly quarter-level milk samples were collected longitudinally from a cohort of 503 primiparous cows from five organic dairy farms during the first 5 weeks after calving. Individual quarter milk samples were pooled in a laboratory using aseptic technique (“lab-pooled”) and subjected to bacterial culture. In the sample set of 2,006 lab-pooled milk samples, 207 (10.3%) were classified as contaminated using a standard definition (i.e., growth of three or more distinct microorganisms). Subsequent culturing of corresponding quarter-level milk samples revealed that many of the contaminated lab-pooled sample results (i.e., 46.7%) were the result of intramammary infections with different pathogens across the quarters, rather than actual contamination within any single quarter (i.e., “true contamination”). The odds of true contamination were lower when the lab-pooled sample exhibited growth of three microorganisms compared to more than 3 microorganisms. Our findings suggest that pooling of quarter samples within a laboratory setting may yield lower rates of contamination compared to those previously reported from samples composited on-farm, but that current cut-offs to define contamination may need to be evaluated for use with lab-pooled samples. Further investigation of use of lab-pooled samples may be warranted to reduce costs while still providing useful scientific insight.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2022 Chris J. Dean. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="noopener noreferrer">Unsplash</a>. Last updated: November 06, 2022. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>